lib/streamlit/testing/v1/app_test.py

```diff
@@ -103,20 +161,29 @@ def __init__(self, script_path: str, *, default_timeout: float):
 
     @classmethod
     def from_string(cls, script: str, *, default_timeout: float = 3) -> AppTest:
-        """Create a runner for a script with the contents from a string.
+        """
+        Create an instance of ``AppTest`` to simulate an app page defined\
+        within a string.
 
-        Useful for testing short scripts that fit comfortably as an inline
-        string in the test itself, without having to create a separate file
-        for it.
+        This is useful for testing short scripts that fit comfortably as an
+        inline string in the test itself, without having to create a separate
+        file for it. The script must be executable on its own and so must
+        contain all necessary imports.
 
         Parameters
         ----------
-        script
+        script: str
             The string contents of the script to be run.
 
-        default_timeout
+        default_timeout: float
             Default time in seconds before a script run is timed out. Can be
-            overridden for individual `.run()` calls.
+            overridden for individual ``.run()`` calls.
+
+        Returns
+        -------
+        AppTest
+            A simulated script runner for testing a script. The script
```

Comment:
Should we do `simulated script runner` -> `simulated Streamlit app` here and in the other two from_ methods as well?

Reply:
And maybe the "the app can be executed via `.run()`"?

Reply:
Oops, missed those when I updated the terminology elsewhere. Thanks.